gsa/src/web/components/dashboard/dashboard.js

```diff
@@ -61,7 +61,11 @@ import withGmp from 'web/utils/withGmp';
 import compose from 'web/utils/compose';
 
 import {getDisplay} from './registry';
-import {getRows} from './utils';
+import {getRows as get_rows} from './utils';
+
+import {useGetSetting} from 'web/pages/tasks/graphql';
+import Setting from 'gmp/models/setting';
+import {convertLoadedSettings} from 'gmp/commands/dashboards';
```

Comment:
Sorry for nitpicking, but could you sort the imports according to our convention? I know, this is a POC, but we will probably use this code later on as well. Therefore, we should tidy it up and make clean commits to save us more work in the future. Better make it nice from the beginning than spending resources on changing stuff later.

Reply:
Of course.